sdk/src/index.ts

```diff
@@ -31,7 +50,7 @@ export class ScryptedDeviceBase extends DeviceBase {
 
   get console() {
     if (!this._console) {
-      this._console = deviceManager.getDeviceConsole(this.nativeId);
+      this._console = deviceManager.getDeviceConsole?.(this.nativeId);
```

Comment:
this should never be nullish, and the signature on this method could be fixed.

Reply:
Fixed